nodejs/week1/avg.js

```diff
@@ -0,0 +1,21 @@
+// console.log(process.argv);
+const myArgs = process.argv.slice(2)
```

Comment:
For an inout like this 
`node avg.js 1 two hello 4`
You progran will skip the strings and will do average of 1 and 4. Not sure if this in intentional or should you return that not valid numbers?

Reply:
As as I understood the task I have filtered all the numbers and then divided that with arrays's length  of numbers.